docs/installation.rst

```diff
@@ -55,6 +55,7 @@ The steps below are the installation instructions for Linux, macOS and Windows.
     conda activate parcels
 
 
+
```

Comment:
Is this extra empty line useful for something?

Reply:
Not at all. Not sure how that crept in